src/components/pages/Announcements/Filter/index.tsx

```diff
@@ -97,7 +97,7 @@ const FilterAnnouncement = ({ closeModal, action }: TFilterAnnouncement) => {
               </CustomField>
             </div>
           </div>
-          <div className="mt-4">
+          <div className={`mt-4 ${!fromDate ? 'pb-4' : ''}`}>
```

Comment:
```suggestion
          <div className="my-4">
```
is there a reason why we can't use this?

Reply:
I have adjusted it